cli/flox/src/commands/activate.rs

```diff
@@ -264,15 +267,16 @@ impl Activate {
             ),
         ]);
 
+        let socket_path = environment.services_socket_path(&flox)?;
         if let TypedManifest::Catalog(manifest) = environment.manifest(&flox)? {
             // default to enabling CUDA
             if manifest.options.cuda_detection != Some(false) {
                 exports.insert("_FLOX_ENV_CUDA_DETECTION", "1".to_string());
             }
 
             if flox.features.services && !manifest.services.is_empty() {
+                supported_environment(&flox, self.environment)?; // Error for remote envs.
```

Comment:
this still causes an error if a remote env has services right?
shouldnt we just ignore services there and print a warning?

Can do in a follow up to make progress on this👍🏼

Reply:
Yeah, per an [existing test](https://github.com/flox/flox/pull/1837/commits/2400a701c007eb250a031695cf52961bdb125c7a#diff-977ce1933722e4c27751f7ba5dce3a3ad5ad067cd1fa16e796c69b23003b7495R350). It's a good point; the thing we really want to prevent is starting services for remote environments (which is a result of the environment variable being set in the activation) so we could wrap this in `if self.start_services {}`.

Reply:
Updated the logic and tests to only block `activate --start-services`